CHANGELOG.mdx

```diff
@@ -16,6 +16,122 @@ https://github.com/RasaHQ/rasa/tree/main/changelog/ . -->
 
 <!-- TOWNCRIER -->
 
+## [2.8.0] - 2021-07-12
```

Comment:
Looks like you need to make sure your base branch is up-to-date. These changes shouldn't be here.